### PR TITLE
Identity/MusicStore/ErrorPage test re-enabling

### LINKED ISSUE #10672: Re-enable DatabaseErrorPage_ShowsError test
This will require either a change to exception wrapping on EF Core, or a change to the filtering in the Database Error Page.

### PR DESCRIPTION
**Pull request by ajcvickers**
Fixes #10672 #7925 #10670

Also, moves MusicStore tests to SQLite for better reliability

### CODE REVIEW

File: src/Identity/EntityFrameworkCore/test/EF.Test/MaxKeyLengthSchemaTest.cs [hunk 1]
```diff
@@ -24,59 +23,73 @@ public MaxKeyLengthSchemaTest(ScratchDatabaseFixture fixture)
...
             {
-                db.Database.EnsureCreated();
+                scope.ServiceProvider.GetRequiredService<VerstappenDbContext>().Database.EnsureCreated();
             }
         }
 
-        [ConditionalFact(Skip = "https://github.com/aspnet/AspNetCore/issues/7925")]
+        // Need a different context type here since EF model is changing by having MaxLengthForKeys
+        public class VerstappenDbContext : IdentityDbContext<IdentityUser, IdentityRole, string>
+        {
```
Comment (HaoK) [inline src/Identity/EntityFrameworkCore/test/EF.Test/MaxKeyLengthSchemaTest.cs]: Random question, what is a `Verstappen`?

**Review by HaoK — APPROVED** [review]

File: src/Identity/EntityFrameworkCore/test/EF.Test/MaxKeyLengthSchemaTest.cs (hunk 1 above)
Comment (ajcvickers) [reply src/Identity/EntityFrameworkCore/test/EF.Test/MaxKeyLengthSchemaTest.cs]: It's just a different way of saying `Max`. And this is a context for testing MaxKeyLength.

Comment (dougbu) [reply src/Identity/EntityFrameworkCore/test/EF.Test/MaxKeyLengthSchemaTest.cs]: A Formula1 inside joke. Nice 😃

Comment (anurse) [reply src/Identity/EntityFrameworkCore/test/EF.Test/MaxKeyLengthSchemaTest.cs]: 🙄

**Pull request merged**